graph/sql/quadstore.go

```
@@ -624,13 +632,10 @@ func (qs *QuadStore) NameOf(v graph.Ref) quad.Value {
 		panic(fmt.Errorf("unexpected token: %T", v))
```

We could also return an error here now.